lib/plugins/threaded-signing/plugin.c

```diff
@@ -21,41 +21,45 @@
 
 /**
  * This signing plugin sets up a worker thread and calls openssl_sign_hash(), from the worker
- * thread, when there is a new hash to sign. The plugin can only handle one signature at a time,
- * that is, there is no queue of hashes to sign. If the latest signature is not completed by the
- * time of a new request, that new hash is not signed.
+ * thread, when there is a new hash to sign. To handle several signatures at the same time, the
+ * plugin has two buffers. One for incomming hashes and another for outgoing signatures.
```

Comment:
I think it would be good to add, in the description, under what conditions the thread is stopped.